app/components/page-components/Chatbot/Poll/ChatbotPollOptionModal.vue.ts

```diff
@@ -0,0 +1,97 @@
+import { Component, Prop, Watch } from 'vue-property-decorator';
+import TextAreaInput from 'components/shared/inputs/TextAreaInput.vue';
+import ListInput from 'components/shared/inputs/ListInput.vue';
+import NumberInput from 'components/shared/inputs/NumberInput.vue';
+import { $t } from 'services/i18n';
+import ValidatedForm from 'components/shared/inputs/ValidatedForm.vue';
+import { EInputType } from 'components/shared/inputs/index';
+
+import { IPollOption } from 'services/chatbot';
+import VFormGroup from 'components/shared/inputs/VFormGroup.vue';
+import ChatbotWindowsBase from '../windows/ChatbotWindowsBase.vue';
+import { debounce } from 'lodash-decorators';
+
+@Component({
+  components: {
+    VFormGroup,
+    TextAreaInput,
+    ListInput,
+    NumberInput,
+    ValidatedForm,
+  },
+})
+export default class ChatbotPollOptionModal extends ChatbotWindowsBase {
+  $refs: {
+    modalForm: ValidatedForm;
+  };
+
+  @Prop({
+    default: {
+      name: null,
+      parameter: null,
+    },
+  })
+  option: IPollOption;
+
+  @Prop() index: number = -1;
```

Comment:
I assume Vue complains here about you're trying to change the prop inside the component instead of using `@Prop({ default: 1})`

Reply:
It's only necessary to use {default: xyz} when dealing with an object. Vue does not complain when using plain data types such as string or number.